sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -587,7 +601,7 @@ export class OfferingValidationModel {
   @HasValidOfferingPeriodForFundedDays(
     studyStartDateProperty,
     studyEndDateProperty,
-    studyPeriodMinLength,
```

Comment:
Have we removed the code related to this function and the respective validation `studyPeriodMinLength` from the `studyEndDate` property?